docker/worker/Dockerfile

```diff
@@ -64,15 +74,20 @@ RUN echo "turbinia ALL = (root) NOPASSWD: ALL" > /etc/sudoers.d/turbinia
 
 RUN pip3 install impacket --no-deps
 
-RUN mkdir -p /opt/loki && chown -R turbinia:turbinia /opt/loki \
-    && cd /opt/loki \
-    && curl -s https://api.github.com/repos/Neo23x0/Loki/releases/latest | sed -n 's/.*"tarball_url": "\(.*\)",.*/\1/p' | xargs -n1 wget -O - -q | tar -xz --strip-components=1 \
-    && pip3 install -r requirements.txt \
-    && sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
-    && sudo chown -R turbinia:turbinia /opt/loki \
-    && git clone https://github.com/Neo23x0/signature-base.git
+RUN cd /opt \
+    && git clone https://github.com/Neo23x0/signature-base.git \
+    && sudo chown -R turbinia:turbinia /opt/signature-base
 
-COPY turbinia/config/rules/*.yar /opt/loki/signature-base/yara/
+COPY turbinia/config/rules/*.yar /opt/signature-base/yara/
+
+RUN mkdir -p /opt/fraken/yara && chown -R turbinia:turbinia /opt/fraken
+COPY --chown=turbinia:turbinia tools/fraken/* /opt/fraken/
+RUN cd /opt/fraken/yara && curl -s https://api.github.com/repos/VirusTotal/Yara/releases/latest | sed -n 's/.*"tarball_url": "\(.*\)",.*/\1/p' | xargs -n1 wget -O - -q | tar -xz --strip-components=1 \
+    && ./bootstrap.sh \
```

Comment:
Current packaged version is 3.9, and up to date is 4.x
This would require researching and pinning an older version of the Golang bindings for the library rather than just fetching the most up to date one upon build